rfcs/0004-leader-rotation.md

```diff
@@ -1,104 +1,29 @@
 # Leader Rotation
 
-The goal of this RFC is to define how leader nodes are rotated in Solana, how rotation may cause forks to arise, and how the converges
-in response.
+The goal of this RFC is to define how leader nodes are rotated in Solana.
```

Comment:
More precisely, it's to define how fullnodes rotate in taking the leader role.

Reply:
@garious why not leaders?  a leader doesn't need to be a full node per se.